### ISSUE TITLE
Bump session timeout to 2+ days

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#544]
**TL;DR**: Increase time until user is automatically logged out slightly to accommodate most cases where families leave a tab open to complete a study later.

**Narrative**
"As a participant, I sometimes find a study that's interesting and get as far as the first few frames, but then realize that my child isn't ready right now and leave the browser tab open until a day or two later. Then we would like to participate, but we have been logged out."

**Acceptance Criteria**
- [ ] `SESSION_COOKIE_AGE` is set to 48+ hours, at least for participants; OR session expiration is set to 48+ hours from now upon beginning participation in a study.

**Implementation Notes**
Will complement changes to address https://github.com/lookit/ember-lookit-frameplayer/issues/166 - this will make it less likely, that will do a better job catching any cases and prompting accordingly.
